content/applications/hr/recruitment/source_analysis.rst

```diff
@@ -0,0 +1,86 @@
+=========================
+Source analysis reporting
+=========================
+
+Reporting is a critical tool used by recruiting departments to gain insights into the entire
+recruitment process.
+
+Determining where applicants come from can provide information about which sources have the best
+results. This information is determined by the *Source Analysis* report. With this data, recruitment
+teams can better pivot their recruiting strategies to gain better applicants, in both quantity and
+quality.
+
+Open report
+===========
+
+To access the *Source Analysis* report, navigate to :menuselection:`Recruitment app --> Reporting
+--> Source Analysis`.
+
+This presents the data for the :icon:`fa-filter` :guilabel:`Last 365 Days Applicant`, in a default
+:icon:`fa-area-chart` :guilabel:`(Graph)` view, showing the amount of applicants by
+:guilabel:`Source`, and further separated by stage (:guilabel:`In Progress` and :guilabel:`Hired`)
+
+Hover the cursor over any column to view the specific numbers fort that column.
+
+.. image:: source_analysis/source-analysis.png
+   :align: center
+   :alt: The default bar chart of the source analysis information.
+
+To view more details, view the :guilabel:`Source Analysis` report in a pivot table. To do so, click
+the :icon:`oi-view-pivot` :guilabel:`(Pivot)` icon in the top-right corner. The data is presented in
+a pivot table, with rows populated by job positions, and columns populated stages.
+
+Source effectiveness report
+===========================
+
+To identify which sources (e.g., job boards, social media, employee referrals, company website)
+produce the most hires, the pivot table view of the :guilabel:`Source Analysis` report can be
+configured to display further details.
+
```

Comment:
Suggestion to move the Pivot view instruction below the section that directs the user to opening Pivot view :)
```suggestion

Source effectiveness report
===========================

To identify which sources (e.g., job boards, social media, employee referrals, company website)
produce the most hires, the pivot table view of the :guilabel:`Source Analysis` report can be
configured to display further details.

To do so, click the :icon:`oi-view-pivot` :guilabel:`(Pivot)` icon in the top-right corner. The data
is presented in a pivot table, with rows populated by job positions, and columns populated stages.
```